app/controllers/favorites_controller.rb

```diff
@@ -20,7 +20,13 @@ def destroy
     favorite.contents.delete(pet.id.to_s)
     flash[:notice] = "#{pet.name} has been removed from your favorites"   
 
-    redirect_to "/pets/#{pet.id}"
+    redirect_back fallback_location: "/favorites/"
```

Comment:
is the "fallback location" for getting the user back to the previous page they were on?

Reply:
Yes, glad to discuss it with you